src/components/structures/MatrixChat.js

```diff
@@ -907,6 +907,11 @@ module.exports = React.createClass({
             console.warn("Failed to set badge count: "+e.message);
         }
         document.title = "Vector"+(notifCount > 0 ? " ["+notifCount+"]" : "");
+
+        if(state === "ERROR") {
+            this.favicon.badge("×");
```

Comment:
I think this has to be in a try block, as per the previous `this.favicon.badge()`?